config/tmux/tmux.conf

```diff
@@ -0,0 +1,83 @@
+# -- General
+
+# override outside terminal
+set -sa terminal-overrides ',xterm-256color:RGB'
+
+# -- Display
+
+# start windows numbering at 1
+set -g base-index 1
+# make pane numbering consistent with windows
+setw -g pane-base-index 1
+# Set terminal title
+set -g set-titles on
+# redraw status line every 10 seconds
+set -g status-interval 10
+
+# -- Navigation
+
+# split current window horizontally
+bind - split-window -v
+# split current window vertically
+bind _ split-window -h
+
+# vim like pane navigation
+bind -r h select-pane -L
+bind -r j select-pane -D
+bind -r k select-pane -U
+bind -r l select-pane -R
+# swap current pane with the next one
+bind > swap-pane -D
+# swap current pane with the previous one
+bind < swap-pane -U
+
+# window navigation
+unbind n
+unbind p
+# select previous window
+bind -r C-h previous-window
+# select next window
+bind -r C-l next-window
+# move to last active window
+bind Tab last-window
+bind t select-layout tiled
+
+# enable mouse navigation
+set -g mouse on
+
+# -- Copy mode
+
+bind Enter copy-mode
+
+run -b 'tmux bind -t vi-copy v begin-selection 2> /dev/null || true'
+run -b 'tmux bind -T copy-mode-vi v send -X begin-selection 2> /dev/null || true'
+run -b 'tmux bind -t vi-copy C-v rectangle-toggle 2> /dev/null || true'
+run -b 'tmux bind -T copy-mode-vi C-v send -X rectangle-toggle 2> /dev/null || true'
+run -b 'tmux bind -t vi-copy y copy-selection 2> /dev/null || true'
+run -b 'tmux bind -T copy-mode-vi y send -X copy-selection-and-cancel 2> /dev/null || true'
+run -b 'tmux bind -t vi-copy Escape cancel 2> /dev/null || true'
+run -b 'tmux bind -T copy-mode-vi Escape send -X cancel 2> /dev/null || true'
+run -b 'tmux bind -t vi-copy H start-of-line 2> /dev/null || true'
+run -b 'tmux bind -T copy-mode-vi H send -X start-of-line 2> /dev/null || true'
+run -b 'tmux bind -t vi-copy L end-of-line 2> /dev/null || true'
+run -b 'tmux bind -T copy-mode-vi L send -X end-of-line 2> /dev/null || true'
+
+# Plugins
+set -g @plugin 'tmux-plugins/tpm'
+set -g @plugin 'tmux-plugins/tmux-sensible'
+set -g @plugin 'tmux-plugins/tmux-urlview'
+set -g @plugin 'tmux-plugins/tmux-yank'
+set -g @plugin 'dracula/tmux'
```

Comment:
are you using dracula?

Reply:
Yeah! It seemed to be the best I could find and also copied it from someone haha